forest/benchmarking/entangled_states.py

```diff
@@ -8,11 +8,12 @@
 from forest.benchmarking.compilation import basic_compile
 
 
-def create_ghz_program(tree: nx.DiGraph):
+def create_ghz_program(tree: nx.DiGraph, skip_measurements = False):
     """
     Create a Bell/GHZ state with CNOTs described by tree.
 
     :param tree: A tree that describes the CNOTs to perform to create a bell/GHZ state.
+    :param skip_measurements: Whether to skip adding MEASURE instructions to the program.
```

Comment:
Why do we need this?

Reply:
I'm not sure it's significance to the program but previously measurements were added and then immediately removed after the `test_create_ghz_program` call, so I thought this would help to avoid assumptions about the program's instruction order https://github.com/rigetti/forest-benchmarking/blob/4c2c3bf94af4926b61e9072ca71b914972de338c/forest/benchmarking/tests/test_entangled_states.py#L13

Reply:
Ah okay